client/extensions/woocommerce/components/d3/base.js

```diff
@@ -0,0 +1,78 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { select as d3Select } from 'd3-selection';
+
+export class D3Base extends Component {
+	static propTypes = {
+		className: PropTypes.string,
+		drawChart: PropTypes.func.isRequired,
+		getParams: PropTypes.func.isRequired,
+	};
+
+	state = {};
+
+	componentDidMount() {
+		window.addEventListener( 'resize', this.handleResize );
```

Comment:
@psealock You're calling `handleResize()` above and on `componentWillUnmount()` but I don't see a `handleResize` method anywhere?

Reply:
Oops. Weird that window resizing still worked. Fixed in https://github.com/Automattic/wp-calypso/pull/22583/commits/2a516676343067f849c0a7af4c09132d403ba2d9